test/fixtures/help

```diff
@@ -11,6 +11,8 @@ Options:
   -e, --file-encoding      Encoding for file <path>, defaults to utf8   [string]
   -h, --help               Shows this help message                     [boolean]
   -m, --multiple           Allows the selection of multiple items      [boolean]
+  -o, --ordered            Allows the selection of multiple ordered items
+                                                                       [boolean]
```

Comment:
Designer itch here 😅 but maybe rephrase it to fit in a single line?

```suggestion
  -o, --ordered            Selects multiple items in order             [boolean]
```

Reply:
Updated